src/bootstrap/src/lib.rs

```diff
@@ -721,6 +725,11 @@ impl Build {
         if self.config.profiler_enabled(target) {
             features.push_str(" profiler");
         }
+        // Generate memcpy, etc.  FIXME: Remove this once compiler-builtins
+        // automatically detects this target.
+        if target.contains("zkvm") {
+            features.push_str(" compiler-builtins-mem");
+        }
```

Comment:
What does this FIXME mean? Are we expecting compiler-builtins to enable `compiler-builtins-mem` feature by default for zkvm targets? If so, I don't think that will ever happen.